config/checkstyle_input_suppressions.xml

```diff
@@ -2659,6 +2661,8 @@
             files="filters[\\/]suppressionsinglefilter[\\/]InputSuppressionSingleFilter.java"/>
   <suppress id="StyleValidationCommentInInputs"
             files="filters[\\/]suppressionxpathfilter[\\/]InputSuppressionXpathFilter.java"/>
+  <suppress id="StyleValidationCommentInInputs"
+            files="filters[\\/]suppressionxpathfilter[\\/]InputSuppressionXpathFilterEscape.java"/>
```

Comment:
Please remove this suppressions.
Add config comment to input file (it is even better to split input file to 2 files for strings and chars)

Reply:
On it.

Reply:
Done